src/compiler/compiler.lisp

```diff
@@ -1733,5 +1733,6 @@
   `(let ((*literal-table* nil)
          (*variable-counter* 0)
          (*gensym-counter* 0)
-         (*literal-counter* 0))
+         (*literal-counter* 0)
+         (*features* (list :jscl :jscl-xc)))
```

Comment:
Will this add `jscl-xc` during regular compilation too?   `jscl-xc` was intended to be there only during bootstrapping.